appengine/redis/server.js

```diff
@@ -0,0 +1,55 @@
+var redis = require("redis");
+var http = require('http');
+var util = require('util');
+var nconf = require('nconf');
+
+// read in keys and secrets.  You can store these in a variety of ways.  
+// I like to use a keys.json  file that is in the .gitignore file, 
+// but you can also store them in environment variables
+nconf.argv().env().file('keys.json');
+
+// Connect to a redis server provisioned over at 
+// redisLabs.  See the README for more info. 
+var client = redis.createClient(
+    nconf.get("redisPort"),
```

Comment:
I would suggest changing this to `nconf.get("redisPort") || "6379",` so `npm start` will work out of the box with a default redis configuration.